src/runtime-rs/crates/runtimes/common/src/sandbox.rs

```diff
@@ -15,6 +18,16 @@ pub struct SandboxNetworkEnv {
     pub network_created: bool,
 }
 
+#[derive(Default, Clone, Debug)]
+pub struct SandboxStatus {
+    pub sandbox_id: String,
+    pub pid: u32,
+    pub state: String,
+    pub info: std::collections::HashMap<String, String>,
+    pub create_at: std::time::Duration,
+    pub exited_at: std::time::Duration,
```

Comment:
Wouldn't it make more sense for these to be `SystemTime` rather than `Duration`?  Also, I can't see them referenced anywhere.

Reply:
Yeah you are right. Since those two fields haven't been referenced, and I removed them now.